*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:09:52 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/MxOnlie
chdir(): No such file or directory [core/uwsgi.c line 2631]
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:10:06 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/MxOnlie
chdir(): No such file or directory [core/uwsgi.c line 2631]
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:11:15 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/MxOnlie
chdir(): No such file or directory [core/uwsgi.c line 2631]
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:12:15 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x1f3a890
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file MxOnlie/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 13827)
spawned uWSGI worker 1 (pid: 13828, cores: 2)
spawned uWSGI worker 2 (pid: 13829, cores: 2)
spawned uWSGI worker 3 (pid: 13830, cores: 2)
spawned uWSGI worker 4 (pid: 13831, cores: 2)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:27:39 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x1295890
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file MxOnlie/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 13989)
spawned uWSGI worker 1 (pid: 13990, cores: 2)
spawned uWSGI worker 2 (pid: 13991, cores: 2)
spawned uWSGI worker 3 (pid: 13992, cores: 2)
spawned uWSGI worker 4 (pid: 13993, cores: 2)
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 07:35:54 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (127.0.0.1:8000).
bind(): Address already in use [core/socket.c line 769]
Thu Aug 30 08:38:00 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 08:38:00 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 08:38:20 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.01:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:46543 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x1fde9a0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file MxOnlie/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
spawned uWSGI master process (pid: 14448)
spawned uWSGI worker 1 (pid: 14449, cores: 2)
spawned uWSGI worker 2 (pid: 14450, cores: 2)
spawned uWSGI worker 3 (pid: 14451, cores: 2)
spawned uWSGI worker 4 (pid: 14452, cores: 2)
spawned uWSGI http 1 (pid: 14453)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 14453)
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 09:07:09 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x21088b0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
failed to open python file MxOnlie/wsgi.py
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 14869)
spawned uWSGI worker 1 (pid: 14870, cores: 2)
spawned uWSGI worker 2 (pid: 14871, cores: 2)
spawned uWSGI worker 3 (pid: 14872, cores: 2)
spawned uWSGI worker 4 (pid: 14873, cores: 2)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/1] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:28 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14873|app: -1|req: -1/2] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:32 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/3] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:35 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/4] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:36 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/5] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:39 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/6] 113.206.182.155 () {44 vars in 730 bytes} [Thu Aug 30 09:07:47 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/7] 113.206.182.155 () {44 vars in 708 bytes} [Thu Aug 30 09:07:48 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/8] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:07:50 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/9] 113.206.182.155 () {44 vars in 708 bytes} [Thu Aug 30 09:07:51 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/10] 223.104.25.4 () {40 vars in 662 bytes} [Thu Aug 30 09:07:53 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/11] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:08:10 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/12] 113.206.182.155 () {44 vars in 708 bytes} [Thu Aug 30 09:08:10 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/13] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:08:12 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/14] 223.104.25.4 () {40 vars in 662 bytes} [Thu Aug 30 09:08:21 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/15] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:08:26 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/16] 223.104.25.4 () {40 vars in 662 bytes} [Thu Aug 30 09:08:27 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/17] 113.206.182.155 () {44 vars in 708 bytes} [Thu Aug 30 09:08:31 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/18] 223.104.25.4 () {40 vars in 662 bytes} [Thu Aug 30 09:11:47 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/19] 223.104.25.4 () {40 vars in 662 bytes} [Thu Aug 30 09:11:49 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/20] 207.148.106.15 () {42 vars in 703 bytes} [Thu Aug 30 09:12:14 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/21] 207.148.106.15 () {44 vars in 703 bytes} [Thu Aug 30 09:12:14 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/22] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:14:48 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/23] 113.206.182.155 () {40 vars in 655 bytes} [Thu Aug 30 09:14:49 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14872|app: -1|req: -1/24] 113.206.182.155 () {42 vars in 708 bytes} [Thu Aug 30 09:15:26 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
--- no python application found, check your startup logs for errors ---
[pid: 14873|app: -1|req: -1/25] 113.206.182.155 () {40 vars in 655 bytes} [Thu Aug 30 09:15:26 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
SIGINT/SIGQUIT received...killing workers...
worker 1 buried after 1 seconds
worker 2 buried after 1 seconds
worker 3 buried after 1 seconds
worker 4 buried after 1 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 09:39:05 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x1c788a0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
Traceback (most recent call last):
  File "MxOnline/wsgi.py", line 16, in <module>
    application = get_wsgi_application()
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/core/wsgi.py", line 13, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/__init__.py", line 27, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/apps/registry.py", line 116, in populate
    app_config.ready()
  File "/root/MxOnline/extra_apps/xadmin/apps.py", line 14, in ready
    self.module.autodiscover()
  File "/root/MxOnline/extra_apps/xadmin/__init__.py", line 49, in autodiscover
    register_builtin_plugins(site)
  File "/root/MxOnline/extra_apps/xadmin/plugins/__init__.py", line 41, in register_builtin_plugins
    [import_module('xadmin.plugins.%s' % plugin) for plugin in PLUGINS if plugin not in exclude_plugins]
  File "/root/MxOnline/extra_apps/xadmin/plugins/__init__.py", line 41, in <listcomp>
    [import_module('xadmin.plugins.%s' % plugin) for plugin in PLUGINS if plugin not in exclude_plugins]
  File "/root/Envs/mxonline/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 665, in exec_module
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/root/MxOnline/extra_apps/xadmin/plugins/themes.py", line 21, in <module>
    class ThemePlugin(BaseAdminPlugin):
  File "/root/MxOnline/extra_apps/xadmin/plugins/themes.py", line 27, in ThemePlugin
    default_theme = static('xadmin/css/themes/bootstrap-xadmin.css')
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/contrib/staticfiles/templatetags/staticfiles.py", line 12, in static
    return _static(path)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/templatetags/static.py", line 166, in static
    return StaticNode.handle_simple(path)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/templatetags/static.py", line 117, in handle_simple
    return staticfiles_storage.url(path)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/utils/functional.py", line 238, in inner
    self._setup()
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/contrib/staticfiles/storage.py", line 504, in _setup
    self._wrapped = get_storage_class(settings.STATICFILES_STORAGE)()
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/contrib/staticfiles/storage.py", line 39, in __init__
    check_settings(base_url)
  File "/root/Envs/mxonline/lib/python3.5/site-packages/django/contrib/staticfiles/utils.py", line 52, in check_settings
    "You're using the staticfiles app "
django.core.exceptions.ImproperlyConfigured: You're using the staticfiles app without having set the required STATIC_URL setting.
unable to load app 0 (mountpoint='') (callable not found or import error)
*** no app loaded. going in full dynamic mode ***
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15260)
spawned uWSGI worker 1 (pid: 15262, cores: 2)
spawned uWSGI worker 2 (pid: 15263, cores: 2)
spawned uWSGI worker 3 (pid: 15264, cores: 2)
spawned uWSGI worker 4 (pid: 15265, cores: 2)
--- no python application found, check your startup logs for errors ---
[pid: 15262|app: -1|req: -1/1] 113.206.182.155 () {42 vars in 709 bytes} [Thu Aug 30 17:39:17 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 15264|app: -1|req: -1/2] 113.206.182.155 () {44 vars in 709 bytes} [Thu Aug 30 17:39:18 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 15265|app: -1|req: -1/3] 113.206.182.155 () {42 vars in 709 bytes} [Thu Aug 30 17:39:21 2018] GET / => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 0)
--- no python application found, check your startup logs for errors ---
[pid: 15265|app: -1|req: -1/4] 113.206.182.155 () {40 vars in 656 bytes} [Thu Aug 30 17:39:21 2018] GET /favicon.ico => generated 21 bytes in 0 msecs (HTTP/1.1 500) 2 headers in 83 bytes (0 switches on core 1)
Thu Aug 30 17:43:36 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 1 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 4 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 3 screams: UAAAAAAH my master disconnected: i will kill myself !!!
Thu Aug 30 17:43:36 2018 - uWSGI worker 2 screams: UAAAAAAH my master disconnected: i will kill myself !!!
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 09:44:38 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 127.0.0.1:8000 fd 4
uwsgi socket 0 bound to TCP address 127.0.0.1:40195 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x130d990
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x130d990 pid: 15374 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15374)
spawned uWSGI worker 1 (pid: 15376, cores: 2)
spawned uWSGI worker 2 (pid: 15377, cores: 2)
spawned uWSGI worker 3 (pid: 15378, cores: 2)
spawned uWSGI worker 4 (pid: 15379, cores: 2)
spawned uWSGI http 1 (pid: 15382)
SIGINT/SIGQUIT received...killing workers...
gateway "uWSGI http 1" has been buried (pid: 15382)
worker 1 buried after 2 seconds
worker 2 buried after 2 seconds
worker 3 buried after 2 seconds
worker 4 buried after 2 seconds
goodbye to uWSGI.
*** Starting uWSGI 2.0.17.1 (64bit) on [Thu Aug 30 09:49:59 2018] ***
compiled with version: 5.4.0 20160609 on 30 August 2018 05:41:09
os: Linux-4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018
nodename: vultr.guest
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /root/MxOnline
writing pidfile to uwsgi.pid
detected binary path: /root/Envs/mxonline/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /root/MxOnline
your processes number limit is 1793
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 127.0.0.1:8000 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 23 2017, 16:37:01)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x22f48a0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 416880 bytes (407 KB) for 8 cores
*** Operational MODE: preforking+threaded ***
WSGI app 0 (mountpoint='') ready in 2 seconds on interpreter 0x22f48a0 pid: 15470 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI master process (pid: 15470)
spawned uWSGI worker 1 (pid: 15472, cores: 2)
spawned uWSGI worker 2 (pid: 15473, cores: 2)
spawned uWSGI worker 3 (pid: 15474, cores: 2)
spawned uWSGI worker 4 (pid: 15475, cores: 2)
[pid: 15473|app: 0|req: 1/1] 113.206.182.155 () {42 vars in 709 bytes} [Thu Aug 30 17:50:04 2018] GET / => generated 28091 bytes in 371 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 15473|app: 0|req: 2/2] 113.206.182.155 () {40 vars in 656 bytes} [Thu Aug 30 17:50:06 2018] GET /favicon.ico => generated 3573 bytes in 9 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 15475|app: 0|req: 1/3] 125.84.179.78 () {40 vars in 666 bytes} [Thu Aug 30 17:51:21 2018] GET / => generated 28091 bytes in 325 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 15475|app: 0|req: 2/4] 125.84.179.78 () {40 vars in 644 bytes} [Thu Aug 30 17:51:24 2018] GET /favicon.ico => generated 3573 bytes in 10 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 15475|app: 0|req: 3/5] 125.84.179.78 () {42 vars in 723 bytes} [Thu Aug 30 17:51:51 2018] GET / => generated 28091 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 15475|app: 0|req: 4/6] 101.89.239.232 () {36 vars in 613 bytes} [Thu Aug 30 17:52:21 2018] GET / => generated 28091 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 1)
